data/virt_autotest/guest_unattended_installation_files/ignition_config_non_encrypted_image.ign

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
Is it the same content with [ignition_config_encrypted_image.ign](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/18488/files#diff-2481e0f900af21e554f247c99cc881f9ce66365b30d5d1e58e42b3c5023815f7)?

Reply:
They are the same at the moment, but it would be better separate them to differentiate encrypted from non-encrypted. More flexible and resilient.

Reply:
Better link one to the other when they are the same, and creates a new one when differentiated.

Reply:
I do not think this is very necessary.